services/can-watchdog/can-watchdog.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+import glob # For expanding wildcard filepaths
+import jeepney # For interactions with dbus
+from subprocess import call
+
+print('CAN Watchtog has started')
+print('Getting all can rx counters')
+rx_paths = glob.glob('sys/class/net/can*/statistics/rx_bytes')
+print(len(rx_paths), ' found: ', rx_paths)
+
+prev_rx = 0
+curr_rx = 0
+
+sleep(5)
+
+while True:
+    for rx in rx_paths
+        with open(rx, 'r') as file:
```

Comment:
Does this `fopen` and `fclose` for each loop?

Reply:
Fixed